spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -299,24 +339,41 @@
       "person_or_org": {
         "name": organization_name,
         "type": "organizational"
-      },
-      "role": {"id": InvenioRdmRecordConverter::ROLE_OTHER}
+      }
     }.with_indifferent_access
   }
 
   describe "#build_creator_contributor_json" do
-    context 'personal record without user in digital hub, with proper name formatting' do
+    context 'personal record without user in digitalhub with proper name formatting' do
```

Comment:
Is another `context` layer possible for `"personal record without user in digitalhub"`?

Reply:
and then below that context the context that changes e.g. `context "with improper name formatting"`